pip/templates/deploy.py

```diff
@@ -77,17 +77,17 @@ def upload_command(repo_type_key, package_file, wheel_file):
     version = version_file.read().strip()
 try:
     new_package_file = dist_dir + "/{package_file}".replace(".tar.gz", "-{}.tar.gz".format(version))
-    new_wheel_file = dist_dir + "/{wheel_file}".replace(".whl", "-{}.whl".format(version))
+    new_wheel_pep491 = dist_dir + "/{wheel_file}".replace("-", "_").replace(".whl", "-{}-py3-none-any.whl".format(version))
```

Comment:
note that they also require `_` to be used for wheel package name itself, instead of `-`. The suffixes must be `-` separated.